storageClient/api.go

```diff
@@ -54,9 +54,10 @@ type DescribeGroupRequest struct {
 
 // ListGroupsRequest look up groups for the client's account by default or optionally filter by parameters such as client ID
 type ListGroupsRequest struct {
-	ClientID  uuid.UUID `json:"client_id"`
-	NextToken int64     `json:"next_token"`
-	Max       int       `json:"max"`
+	ClientID   uuid.UUID `json:"client_id"`
```

Comment:
since this is a get where params are only passed in the url, none of this struct's fields require json tags as we never try to encode this to json.